stage-file-proxy.php

```diff
@@ -1,39 +1,36 @@
 <?php
 /*
 	Plugin Name: Stage File Proxy
-	Plugin URI: http://alleyinteractive.com/
+	Plugin URI: https://www.alley.com
 	Description: Get only the files you need from your production environment. Don't ever run this in production!
-	Version: 1.1.0
+	Version: 2023
```

Comment:
This assumes 1 release per calendar year doesn't it?

Reply:
I suppose it could be read that way. On the other hand, the plugin hasn't ever actually had any tags or releases, as far as I'm aware. No one installed version `100` before and wouldn't have a way to install version `2023` now. It's only ever run off trunk. If push really came to shove, the version could be bumped to `2023.2` or given a proper release with a proper semantic version number.